### ISSUE TITLE
remove examples from site-packages

### ISSUE THREAD

**Issue by GarryMorrison** [GarryMorrison/Semantic-DB#17]
There is no point to keeping copies of sw/ and graph/ examples in site-packages/. Is there an easy way to prevent their install?

**Comment by GarryMorrison** [comment]
docs/ doesn't seem to be in site-packages/ so if we move examples/ back to the root directory, then we should be done? @gavinfernandes2012 
And then we need to update the path in our new setup.py in run().
```
            # copy example files to destination directory:
            for file in glob.glob(dir + "/semantic_db/examples/sw/*"):

            # copy graph files to destination directory:
            for file in glob.glob(dir + "/semantic_db/examples/graph/*"):
```
to:
```
            # copy example files to destination directory:
            for file in glob.glob(dir + "/examples/sw/*"):

            # copy graph files to destination directory:
            for file in glob.glob(dir + "/examples/graph/*"):
```

**Comment by gavinfernandes2012** [comment]
Yeah moving examples to the root directory will prevent installation.
And when you talk about changing those lines, in which setup.py file do you mean to change them?
I mean my setup.py doesn't have those lines, and neither does the new one you sent me.
Also one more thing, do you want examples to be copied to a user specified directory upon install?

**Comment by GarryMorrison** [comment]
Yeah, it is in the setup.py that I sent you. I just checked.

"do you want examples to be copied to a user specified directory upon install?"

Doesn't the new setup.py do that?

**Comment by gavinfernandes2012** [comment]
Regarding the new setup.py. I think we should publish to pypi just as a test, and then try installing. I'm not sure, but all that trouble with wheels and install options might prove to be a problem. If it works then I'm all for it.
